pom.xml

```diff
@@ -315,6 +310,11 @@
   </build>
 
   <repositories>
+    <repository>
+      <id>maven-central</id>
```

Comment:
Are you sure about that? I am afraid that we can miss some transitive dependencies not being productized but pulled from maven central

Reply:
This is a community repository. Some plugins were not downloadable only from the JBoss repo.

Reply:
Yes, that is true if a dependency is productized in Red Hat the artifacts would receive different version. Though
For example:
If your upstream project contain a dependency that was decided to be excluded in product - 
Then prod team will provide your project with the profiled sources excluding this dependency and do not provide the dependency productized. In that case - if you add the community repository and there would be a bug in the profile you would not be able to hit that. It will work as usual and download the upstream dependency from the community. WDYT?

Reply:
That should be solved on the productization level, not in the upstream community repository like this. E.g. with using different settings.xml file during builds which removes Maven Central. Similarly, there is Maven Central e.g. in Drools repository https://github.com/kiegroup/drools/blob/main/pom.xml#L46

Reply:
Agree but we want to check that, right?

In productized sources it is left empty 
 <repositories> </repositories>
  <pluginRepositories></pluginRepositories>

Reply:
Checked no affection on the tests